src/fields/navLink/components/Description.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+
+import { useField } from '@payloadcms/ui'
+import { FieldDescriptionClientComponent } from 'payload'
+
+export const Description: FieldDescriptionClientComponent = ({ path }) => {
+  const linkPath = path.split('.').slice(0, -1).join('.')
+  const linkTypePath = [linkPath, 'type'].join('.')
```

Comment:
This is a bit of wizardry - could you add a comment for what the expected input (`path`) and the expected output (for `useField`) are to help describe what the transforms are doing?

```
// transform input paths like `` to their field identifier ``
```